README.md

```diff
@@ -115,6 +116,21 @@ npm run e2e.live
 ```
 You can learn more about [Protractor Interactive Mode here](https://github.com/angular/protractor/blob/master/docs/debugging.md#testing-out-protractor-interactively)
 
+# Progressive apps
+
+`angular2-seed` supports progressive apps with [angular/progressive](https://github.com/angular/progressive).
```

Comment:
Same comment as earlier. https://github.com/angular/progressive specifically tries to help with building https://developers.google.com/web/progressive-web-apps